pages/add-a-company.tsx

```diff
@@ -0,0 +1,58 @@
+import Layout from "../components/Layout";
+import FormField from "../components/Globals/FormField";
+import { BackButton, SubmitButton } from "../components/Globals/FormButtons";
+
+const pageDescription =
+  "Add a company to the Code Youngstown local tech company directory.";
+
+export default function AddACompany() {
+  const BASE_URL = process.env.NEXT_PUBLIC_VERCEL_URL;
+  return (
+    <Layout page="Add Company Form" description={pageDescription}>
```

Comment:
I think the name of this page should probably be `Add a Company` instead of `Add Company Form` for SEO purposes